api/graphql/schema.graphql

```diff
@@ -1609,18 +1608,18 @@ type Mutation {
     @guard
     @canModel(ability: "create")
     @injectNow(name: "submitted_at")
-  updatePoolCandidateAsAdmin(
+  updatePoolCandidateStatusAndExpiry(
     id: UUID!
-    poolCandidate: UpdatePoolCandidateAsAdminInput! @spread
+    poolCandidate: UpdatePoolCandidateStatusAndExpiryInput! @spread
   ): PoolCandidate
     @update
     @guard
-    @canFind(ability: "updateAsAdmin", find: "id", injectArgs: true)
+    @canFind(ability: "updateStatus", find: "id", injectArgs: true)
   # Return a boolean to prevent automatically updating the client's cache
   togglePoolCandidateBookmark(id: ID!): Boolean
     @guard
     @canFind(
-      ability: "updateAsAdmin"
+      ability: "updateStatus"
```

Comment:
Could you create a new method in the policy for updateBookmark? Even if we leverage another permission there, that seems like the right level of abstraction.

Reply:
https://github.com/GCTC-NTGC/gc-digital-talent/pull/9696/commits/03944f36cd124ba0d47b0a5bf28bf8687aaccac7